.ci/bump-stack-version.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Given the stack version this script will bump the version.
+#
+# This script is executed by the automation we are putting in place
+# and it requires the git add/commit commands.
+#
+# Parameters:
+#	$1 -> the version to be bumped. Mandatory.
+#	$2 -> whether to create a branch where to commit the changes to.
+#		  this is required when reusing an existing Pull Request.
+#		  Optional. Default true.
+#
+set -euo pipefail
+MSG="parameter missing."
+VERSION=${1:?$MSG}
+CREATE_BRANCH=${2:-true}
+
+OS=$(uname -s| tr '[:upper:]' '[:lower:]')
+
+if [ "${OS}" == "darwin" ] ; then
+	SED="sed -i .bck"
+else
+	SED="sed -i"
+fi
+
+echo "Update stack with version ${VERSION}"
+${SED} -E -e "s#(DefaultStackVersion) = \"[0-9]+\.[0-9]+\.[0-9]+(-[a-f0-9]{8})?#\1 = \"${VERSION}#g" internal/install/stack_version.go
```

Comment:
What's the file with the stack version used in the tests? If any. Otherwise we don't need this automation

Reply:
It's always the current one tagged with DefaultStackVersion, e.g. 7.13.0-SNAPSHOT.

Reply:
@v1v 

Bumping up the PR, what's the decision/plan here? Would you like to discuss it first?